### PR TITLE
Migrate to InSpec

### PR DESCRIPTION
**Pull request by atomic111**

### CODE REVIEW

File: inspec.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+name: postgres-hardening
```
Comment (chris-rock) [inline inspec.yml]: we should align this with our new wording: `postgres-baseline`

File: inspec.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+title: Hardening Framework Postgres Hardening Test Suite
+maintainer: Hardening Framework Team
+copyright: Hardening Framework Team
+copyright_email: hello@dev-sec.io
+license: Apache 2 license
+summary: Test-suite for best-practice postgres hardening
+version: 0.1.0
```
Comment (chris-rock) [inline inspec.yml]: I think we should bump it to version 2.0.0 since 1.x is the Serverspec-based implementation

File: controls/postgres_spec.rb [hunk 3]
```diff
@@ -0,0 +1,268 @@
...
+# author: Christoph Hartmann
+# author: Dominik Richter
+# author: Patrick Muench
+
+title 'Postgres server config'
+
+# attributes
+
+DB_USER = attribute(
+  'db_user',
```
Comment (chris-rock) [inline controls/postgres_spec.rb]: should we just stick to `user` and `password`?

**Comment by chris-rock** [comment]
@atomic111 should we go forward and merge this PR? We could fix issues in further PRs? Do you have a list of points that is missing before the migration is complete?

**Comment by chris-rock** [comment]
This profile works well with the latest version of InSpec. It depends on https://github.com/chef/inspec/pull/826 and https://github.com/chef/inspec/pull/1395 to work properly:

screen shot 2017-01-03 at 6 53 15 pm

Awesome work @atomic111

**Comment by atomic111** [comment]
@chris-rock thank you for the bugfix and now we can merge it. great!!!

**Comment by chris-rock** [comment]
Thank you for this huge update @atomic111

**Pull request merged**